### DISCUSSION TITLE
2021.06.08

### DISCUSSION THREAD

**Discussion by malaheaven** [malaheaven/issue-tracker#7] [General]
#### 회의
1. 이모티콘 배제(시간이 되면 적용)
2. 필터에 대한 부분(ios는 ios가, fe도 fe가)

**Comment by Settpark** [comment]
iOS: 금일 OAtuh 구현 준비 및 기본 UI 작성 했습니다.

**Comment by 17-sss** [comment]
**FE**
- 로그인 / 회원가입 페이지를 만들기 위한 이번에 새로운 기술을 접하고, 적용하려 노력했습니다.
- 추가적인 API에 관한 사항들을 의논하였습니다.

**Comment by malaheaven** [comment]
### BE
- api 협의
- mock api 제작
